react/features/filmstrip/components/web/Toolbar.js

```diff
@@ -46,6 +47,9 @@ class Toolbar extends Component<Props> {
                 <AudioMuteButton
                     tooltipPosition = 'left'
                     visible = { this._shouldShowButton('microphone') } />
+                <PresenterMuteButton
```

Comment:
In filmstrip-only mode, where this is used, so many buttons are not going to fit.

Reply:
Removed the PresenterMuteButton from the filmstrip-only mode.